### PR TITLE
Add support link to footer

### PR DESCRIPTION
**Pull request by georgeschena**
Add Support footer link

Screenshot 2022-04-06 at 13 22 32

### CODE REVIEW

File: config/nunjucks/globals.js [hunk 1]
```diff
@@ -8,18 +8,27 @@ const {
...
 const { mountpath: componentsUrl } = require('../../app/components')
 const i18n = require('../i18n')
 const logger = require('../logger')
 const { manifest: manifestPath } = require('../paths')
 
 let webpackManifest = {}
+
+const supportLink = IS_PRODUCTION
+  ? 'https://support.hmpps.service.justice.gov.uk/feedback-and-support/book-secure-move'
+  : 'https://support-dev.hmpps.service.justice.gov.uk/feedback-and-support/book-secure-move'
```
Comment (thomasleese) [inline config/nunjucks/globals.js:22]: I'm thinking we should move this logic in to the `config/index.js` file, in a similar way to how we have `FEEDBACK_URL` coming directly from the config.

It's not a big deal, but it means we can use this in multiple places if we needed to without needing to extra the logic out of this file.

**Review by thomasleese — APPROVED** [review]

File: config/nunjucks/globals.js (hunk 1 above)
Comment (georgeschena) [reply config/nunjucks/globals.js:22]: Good shout

**Review by Thource — APPROVED** [review]

**Review by thomasleese — APPROVED** [review]

**Comment by ElizabethGoddard** [comment]
Hi George, the review button has gone for me but I am happy to approve this. Thanks, Lizzie

**Comment by ElizabethGoddard** [comment]
Actually @georgeschena I just realised I can't check where the actual link goes from a screenshot - could you let me re-review once this is in staging? Thanks!

**Pull request merged**